yt/chyt/controller/internal/jupyt/configs.go

```diff
@@ -0,0 +1,46 @@
+package jupyt
+
+import (
+	"context"
+	"encoding/json"
+	"go.ytsaurus.tech/yt/go/ypath"
+	"go.ytsaurus.tech/yt/go/yt"
+)
+
+type YTServerConfig struct {
```

Comment:
The config name doesn't make sense to me.
In CHYT the binary was initially called `ytserver-clickhouse`, that's why in the chyt controller the config is called `ytServerClickHouseConfig`.

Also, we don't need to export this type outside of this package, so it's name should start with lower case letter.

`jupytConfig`? `jupytServerConfig`? `jupYTServerConfig`?